src/pages/home/report/ReportActionsView.js

```diff
@@ -46,8 +62,15 @@ class ReportActionsView extends React.Component {
         this.scrollToListBottom = this.scrollToListBottom.bind(this);
         this.recordMaxAction = this.recordMaxAction.bind(this);
         this.onVisibilityChange = this.onVisibilityChange.bind(this);
-        this.sortedReportActions = this.updateSortedReportActions();
+        this.setUpUnreadActionIndicator = this.setUpUnreadActionIndicator.bind(this);
+        this.updateSortedReportActions = this.updateSortedReportActions.bind(this);
+
+        this.sortedReportActions = [];
```

Comment:
Before, we had `this.sortedReportActions = this.updateSortedReportActions()` in the constructor, and we were also calling `this.updateSortedReportActions()` before the render's `return` statement (just like it is still being called right now).

If we just leave it in the constructor, it doesn't work, which can be quickly seen when we comment out the call before the return statement. Because of that, I felt like calling it in the constructor was not efficient, since we were calling it afterward anyway, and decided to change it to simply initialize an empty array.

Definetly don't need the `bind` tho.